lib/sort.js

```diff
@@ -1,12 +1,11 @@
 'use strict';
 
-const sortComparePriority = (
-  // Compare for array.sort with priority
-  priority, // array of strings with priority
-  s1, s2 // string, to compare
-  // Returns: number
-  // Example: files.sort(common.sortComparePriority)
-) => {
+// Compare for array.sort with priority
+//   priority - <string[]>, with priority
+//   s1, s2 - <string>, to compare
+// Returns: <number>
+// Example: files.sort(common.sortComparePriority)
```

Comment:
I'd suggest to always add blank line between 'function description, args, returns, throws' and 'examples'.